NetworkingSampleApp/NetworkingSampleApp/Scenes/Upload/UploadsView.swift

```diff
@@ -61,3 +47,88 @@ struct UploadsView: View {
         .navigationTitle("Uploads")
     }
 }
+
+private extension UploadsView {
+    var singleUpload: some View {
+        Section("Single") {
+            Button("Photo") { isPhotosPickerPresented = true }
+                .photosPicker(
+                    isPresented: $isPhotosPickerPresented,
+                    selection: $selectedPhotoPickerItem,
+                    matching: .images
+                )
+                .onChange(of: selectedPhotoPickerItem) { photo in
+                    Task {
+                        if let data = try? await photo?.loadTransferable(type: Data.self) {
+                            await viewModel.uploadImage(
+                                data,
+                                fileName: selectedPhotoPickerItem?.supportedContentTypes.first?.preferredFilenameExtension
+                            )
+                        }
```

Comment:
I'd put this piece of code in the VM, sending the photo as param

Reply:
This has been changed in the base branch based on your comment there. It's not been merged here yet.

Reply:
This has been changed in the base branch based on your comment there. It's not been merged here yet.